ddtrace/contrib/django/patch.py

```diff
@@ -106,17 +106,19 @@ def _set_request_tags(django, span, request):
 
     user = getattr(request, "user", None)
     if user is not None:
-        if hasattr(user, "is_authenticated"):
-            span.set_tag("django.user.is_authenticated", user_is_authenticated(user))
-
-        uid = getattr(user, "pk", None)
-        if uid:
-            span.set_tag("django.user.id", uid)
-
-        if config.django.include_user_name:
-            username = getattr(user, "username", None)
-            if username:
-                span.set_tag("django.user.name", username)
+        try:
+            if hasattr(user, "is_authenticated"):
+                span.set_tag("django.user.is_authenticated", user_is_authenticated(user))
+            uid = getattr(user, "pk", None)
+            if uid:
+                span.set_tag("django.user.id", uid)
+
+            if config.django.include_user_name:
+                username = getattr(user, "username", None)
+                if username:
+                    span.set_tag("django.user.name", username)
+        except django.core.exceptions.ImproperlyConfigured:
```

Comment:
Since the error can only be raised on line 111 we should wrap that line specifically with the `try`..`except`.

Reply:
I initially wrapped that line specifically, but it ended up raising the same error on the rest of the `getattr` calls to the user, so I ended up wrapping all of the lines relating to the user object. Is that okay?

Reply:
Oh interesting! In that case then I think it's worthy of a comment so a passerby in the future will know as well 🙂